runtime.go

```diff
@@ -7,17 +7,25 @@ import (
 	"time"
 )
 
-func getNumFiles(dir string) int {
-	f, err := os.Open(dir)
+func getNumFiles(dir string) (n int, err error) {
```

Comment:
added an error to the response so we could better account for and log potential errors with this (fixes for `errcheck` linter)